src/lib/utils/index.ts

```diff
@@ -113,3 +114,17 @@ export async function forChange(
 export function bnSum(amounts: string[]): BigNumber {
   return amounts.reduce((a, b) => bnum(a).plus(b), bnum(0));
 }
+
+/**
+ *  creates a sentence readable list for an array for string
+ *  e.g. 'UBT, DSB and BAL' for [UBT, DSB, BAL]
+ */
+export function formatWordListAsSentence(words: string[]) {
```

Comment:
This should have a test to ensure it behaves as you expect in different cases.

Reply:
> > @pkattera @garethfuller Another big question is when the user enters in a price for an unknown token, if they decide to refresh the page or come back at a later time. Should the price they entered remain as the price they entered in before? What if the 'actual' price of that asset has changed since that last entry time and they are prone to being rekt as a result?
> 
> Good question. I think we should probably invalidate custom prices if they leave the flow. It looks like that's how it works already? The one other thing I think should be possible is if they want to update the price after entering it for the first time. There doesn't seem to be a mechanism for that?

No, but I will discuss with pon

Reply:
added those tests